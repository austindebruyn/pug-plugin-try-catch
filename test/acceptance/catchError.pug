.foo
  try
    h1= func()
  catch
    h1= goodfunc()
