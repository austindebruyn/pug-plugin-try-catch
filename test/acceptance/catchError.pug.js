'use strict';

module.exports = {
  func: function () {
    throw new Error();
  },
  goodfunc: function () {
    return 'ok!';
  }
};
