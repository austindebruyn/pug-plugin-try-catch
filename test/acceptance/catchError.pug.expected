<div class="foo"><h1>ok!</h1></div>
